data/json/npcs/scrap trader/scrap trader missions.json

```diff
@@ -0,0 +1,135 @@
+[
+{
+    "id": "MISSION_SCRAP_TRADER_GET_25_MAGNETRON",
+    "type": "mission_definition",
+    "name": "Find 25 Magnetrons",
+    "description": "Obtain 25 magnetrons in exchange for credit with the scrap merchant.",
+    "goal": "MGOAL_FIND_ITEM",
+	"item": "magnetron",
+	"count": 25,
+    "difficulty": 2,
+    "value": 10000,
+	"origins": [ "ORIGIN_OPENER_NPC" ],          
+    "followup": "MISSION_SCRAP_TRADER_GET_75_CHARCOAL",  
+        "end": {
+           "effect": [ { "npc_add_var": "specific_metal_tradeable", "type": "dialogue", "context": "trade", "value": "yes" } ]
+               }, 
+	"has_generic_rewards": true,
+	"dialogue": {
+      "describe": "I need help...",
+	  "offer": "Since buisness is increasing around here, I've decided to start selling more specialized metal. Problem is, I can't seperate the metal out. Fortunately, I found some old schematics on a machine for this sort of thing. If you could get me 20 magnetrons, I can melt down the sorted stuff into bars. I can quickly pay you in raw materials.",

```

Comment:
```suggestion
	  "offer": "Since business is increasing around here, I've decided to start selling more specialized metal. Problem is, I can't separate the metal out. Fortunately, I found some old schematics on a machine for this sort of thing. If you could get me 20 magnetrons, I can melt down the sorted stuff into bars. I can quickly pay you in raw materials.",

```